unittests/test_modules.py

```diff
@@ -35,7 +35,7 @@ def modules_system(request, monkeypatch):
 
 
 def test_searchpath(modules_system):
-    if modules_system.name == 'nomod':
+    if modules_system.name in ['nomod', 'spack']:
```

Comment:
@victorusu Are we sure that Spack does not support any sort of module path concept?